script.js

```diff
@@ -1,81 +1,119 @@
+      gameData.textContent = message;
+      resultDiv.appendChild(gameData);
       humanScore++;
-      console.log(`Computer score: ${computerScore} \n\Human score: ${humanScore}`);
+      message = `Computer score: ${computerScore} \n\Human score: ${humanScore}`;
+      console.log(message);
+      gameData.classList.add("gameData");
+      gameData.textContent = message;
+      resultDiv.appendChild(gameData);
     } else if (humanChoice == computerChoice) {
-      console.log("Tie! None gets a point");
-    } else console.log("Error, combination not yet added");
+      message = "Tie! None gets a point";
+      console.log(message);
+      gameData.classList.add("gameData");
+      gameData.textContent = message;
+      resultDiv.appendChild(gameData);
+    } else {
+      message = "Error, combination not yet added";
+      console.log(message);
+      gameData.classList.add("gameData");
+      gameData.textContent = message;
+      resultDiv.appendChild(gameData);
+    }
   }
-  // Play game
-  const humanSelection = getHumanChoice();
+function playGame(humanButtonSelection) {
+  playRound();
+
+  const humanSelection = humanButtonSelection;
   const computerSelection = getComputerChoice();
-  // Print choices
-  console.log(`Computer chose ${computerSelection} \n\Human chose ${humanSelection}`);
+
+  let message = `Computer chose ${computerSelection} \n\Human chose ${humanSelection}`;
+  console.log(message);
+  gameData.classList.add("gameData");
+  gameData.textContent = message;
+  resultDiv.appendChild(gameData);
+
   playRound(humanSelection, computerSelection);
+  if (humanScore >= 5 || computerScore >= 5) {
+    if (humanScore > computerScore) {
+      message = "Human won the game, congratulations!";
+      console.log(message);
+      gameData.classList.add("gameData");
+      gameData.textContent = message;
+      resultDiv.appendChild(gameData);
+    }
+    else if (computerScore > humanScore) {
+      message = "Computer won the game, better luck next time human!";
+      console.log(message);
+      gameData.classList.add("gameData");
+      gameData.textContent = message;
+      resultDiv.appendChild(gameData);
+    }
+    else {
+      message = "It's a tie. Give it another go to get a winner!";
+      console.log(message);
+      gameData.classList.add("gameData");
+      gameData.textContent = message;
+      resultDiv.appendChild(gameData);
+    }
   }
-  if (humanScore > computerScore)
-    console.log("Human won the game, congratulations!");
-  else if (computerScore > humanScore)
-    console.log("Computer won the game, better luck next time human!");
-  else console.log("It's a tie. Give it another go to get a winner!");
 }
-playGame();
-// console.log("Game end");
\ No newline at end of file
+
+
+function onRockClick(){
+  playGame("rock");
+}
+
+function onPaperClick(){
+  // console.log("Paper btn");
+  playGame("paper");
+}
+
+function onScissorsClick(){
+  playGame("scissor");
+}
```

Comment:
here you can simplify further by using `playGame("X")` directly in the html and remove all of those